bin/webpack.js

```diff
@@ -399,6 +405,12 @@
 				}
 			});
 
+			ifArg("info-verbosity", function(value) {
+				if (!["none", "info", "verbose"].includes(value))
```

Comment:
@ev1stensberg can we keep this check until schema lands so there is no misuse?

Reply:
Could you create an issue for it? Mergeable after

Reply:
done #249 @ev1stensberg